clash-cores/src/Clash/Cores/Xilinx/BlockRam.hs

```diff
@@ -7,6 +7,8 @@
 -}
 
 -- See [Note: eta port names for tdpbram]
+{-# LANGUAGE CPP #-}
+
 {-# OPTIONS_GHC -fno-do-lambda-eta-expansion #-}
```

Comment:
This one is a bit unfortunate...

Reply:
Ah.. I'll change it before merging!

Reply:
I've tested the code base for all combinations of a single-line Haskell comment immediately preceding the `LANGUAGE CPP` line and this is the only culprit. The other comments pertain to the whole file, and are thus in their right place.

I had to brush up on my `awk` skills (you could say I had become a bit awkward at it?), but this is the script and its output:
<details>

```
clash-compiler$ find * -type f -print0 | xargs -0 gawk '/^--/{found=1; comment=$0; next} found && /^{-# LANGUAGE CPP #-}/{ print FILENAME; print comment ; print ; print "" } { found=0 }'
clash-cores/src/Clash/Cores/Xilinx/BlockRam.hs
-- See [Note: eta port names for tdpbram]
{-# LANGUAGE CPP #-}

tests/shouldwork/Basic/Replace.hs
-- See: https://github.com/clash-lang/clash-compiler/issues/365
{-# LANGUAGE CPP #-}

tests/shouldwork/Vector/FirOddSize.hs
-- see https://github.com/clash-lang/clash-compiler/issues/383
{-# LANGUAGE CPP #-}

tests/shouldwork/Vector/IndexInt2.hs
-- which generates special verilog when the index is constant.
{-# LANGUAGE CPP #-}

tests/shouldwork/Numbers/DivideByZero.hs
-- Test that the compiler doesn't crash on any DivideByZero exceptions
{-# LANGUAGE CPP #-}
```
</details>